extra/TestPlugin/build.gradle.kts

```diff
@@ -9,3 +9,7 @@ repositories {
 dependencies {
 	compileOnly("io.papermc.paper:paper-api:${rootProject.property("paper.api.full-version")}")
 }
+java {
+    sourceCompatibility = JavaVersion.VERSION_19
+    targetCompatibility = JavaVersion.VERSION_19
+}
```

Comment:
why was this added

Reply:
Too fast, it's a mistake on my part, I didn't see it during the push, sorry